### DISCUSSION TITLE
签到不能更新模板了，请求大神帮忙解决，万分感谢

### DISCUSSION THREAD

**Discussion by zzylhy** [qd-today/templates#700] [Q&A] [answered]
开始更新...
-----开始更新 qd-today 模板仓库-----
仓库地址: https://raw.githubusercontent.com/qd-today/templates/master
更新失败, 原因: Cannot connect to host raw.githubusercontent.com:443 ssl:False [Try again]
错误代码：4006,
错误原因：Update failed, please check failure reason.
现在就是这样的，不管点多少都是这样，请问下大神们，怎么解决啊~

**Comment by a76yyyy** [comment, marked as answer]
https://github.com/qd-today/qd/issues/537#issuecomment-2198592998
